lunabot_embedded/src/teensy_driver_node.cpp

```diff
@@ -16,12 +16,60 @@ extern "C" {
 #include "pb_encode.h"
 }
 
+using namespace std;
+
 #define BUF_SIZE 64
 
+#define DEP_GEAR_RATIO (1. / 7.125)
+
+#define MAX_DIFF 15
+
+float to_rad(float deg) { return deg * M_PI / 180.0; }
+float u_mod(float n, float base) {
+    float m = fmod(n, base);
+    return (m >= 0) ? m : m + base;
+}
+float ang_delta(float deg, float prev_deg) {
```

Comment:
This could also go in a helper function file?

Reply:
yeah. theoretically, preprocessing should be done in the c++ and only output in metric units via the RobotState/RobotSensor msg, so we can add this to a helper function

Reply:
Added to https://github.com/PurdueLunabotics/purdue_lunabotics/pull/43/files#diff-6004914c5cf62632d156dd6e43a863628372558461c7a09dce76993368baf5e4